docs/_guide/styles.md

```diff
@@ -213,13 +268,14 @@ Two best practices for working with custom elements are:
 
 See [Custom Element Best Practices](https://developers.google.com/web/fundamentals/web-components/best-practices) for more information.
 
-## Styling elements in a shadow root
+#### Styling elements in the host's shadow root {#children}
```

Comment:
I think `#children` is the wrong fragment. Children should refer to light DOM children.